gc/base/standard/ParallelGlobalGC.cpp

```diff
@@ -1667,7 +1667,7 @@ MM_ParallelGlobalGC::reportGCIncrementEnd(MM_EnvironmentBase *env)
 	}
 
 	stats->_endTime = omrtime_hires_clock();
-
+	stats->_stallTime = _extensions->globalGCStats.markStats.getStallTime() + _extensions->globalGCStats.workPacketStats.getStallTime() + _extensions->globalGCStats.sweepStats.idleTime;
```

Comment:
We should introduce `getTotalStalltime()` for `globalGCStats` to sum these up and return it